backend/app/auth/service.py

```
@@ -116,6 +120,12 @@ async def create_user_with_email(self, email: str, password: str, name: str) ->
                 status_code=status.HTTP_400_BAD_REQUEST,
                 detail="User with this email already exists"
             )
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)